src/scene/vcModel.cpp

```diff
@@ -195,37 +197,6 @@ void vcModel::AddToScene(vcState *pProgramState, vcRenderData *pRenderData)
 
 void vcModel::OnNodeUpdate(vcState *pProgramState)
 {
-  udDouble3 *pPosition = nullptr;
-  double scale;
-  udDouble3 euler;
-
-  if (m_pNode->geomCount != 0)
-  {
-    if (m_pCurrentZone != nullptr)
-    {
-      vcProject_FetchNodeGeometryAsCartesian(&pProgramState->activeProject, m_pNode, *m_pCurrentZone, &pPosition, nullptr);
-    }
-    else if (m_pPreferredProjection != nullptr)
-    {
-      vcProject_FetchNodeGeometryAsCartesian(&pProgramState->activeProject, m_pNode, *m_pPreferredProjection, &pPosition, nullptr);
-      m_pCurrentZone = (udGeoZone*)udMemDup(m_pPreferredProjection, sizeof(udGeoZone), 0, udAF_None);
-    }
-    else
-    {
-      vcProject_FetchNodeGeometryAsCartesian(&pProgramState->activeProject, m_pNode, pProgramState->geozone, &pPosition, nullptr);
-      m_pCurrentZone = (udGeoZone*)udMemDup(&pProgramState->geozone, sizeof(udGeoZone), 0, udAF_None);
-    }
-
-    vdkProjectNode_GetMetadataDouble(m_pNode, "transform.rotation.y", &euler.x, 0.0);
-    vdkProjectNode_GetMetadataDouble(m_pNode, "transform.rotation.p", &euler.y, 0.0);
-    vdkProjectNode_GetMetadataDouble(m_pNode, "transform.rotation.r", &euler.z, 0.0);
-    vdkProjectNode_GetMetadataDouble(m_pNode, "transform.scale", &scale, udMag3(m_baseMatrix.axis.x));
-
-    m_sceneMatrix = udDouble4x4::translation(m_pivot) * udDouble4x4::rotationYPR(UD_DEG2RAD(euler), *pPosition) * udDouble4x4::scaleUniform(scale) * udDouble4x4::translation(-m_pivot);
-  }
-
-  udFree(pPosition);
```

Comment:
These aren't being loaded from disk? That will mean that models aren't saved/loaded correctly.